src/pydash/helpers.py

```diff
@@ -111,7 +111,7 @@ def iterator(obj):
     elif isinstance(obj, Iterable):
         return enumerate(obj)
     else:
-        return iteritems(getattr(obj, '__dict__', {}))
+        return iteritems(dict((attribute, getattr(obj, attribute)) for attribute in dir(obj) if not attribute.startswith('__')))
```

Comment:
This will end up selecting attributes that are meant to be excluded like methods. The intention around using `obj.__dict__` was to only return attributes set on a class. It's not very sophisticated and misses things like properties, but that was intentional.

For those cases, one can pass in a custom iteratee that would select the desired attributes from the class.